v21.1/upgrade-cockroach-version.md

```diff
@@ -81,11 +81,23 @@ Verify the overall health of your cluster using the [DB Console](ui-overview.htm
 
 ### Check decommissioned nodes
 
-Check the `membership` field in the [output of `cockroach node status --decommission`](cockroach-node.html). Nodes with `decommissioned` membership are fully decommissioned, while nodes with `decommissioning` membership have not completed the process. If there are `decommissioning` nodes in your cluster, this will block the upgrade.
+If your cluster contains partially-decommissioned nodes, they will block an upgrade attempt.
 
-**Before upgrading from v20.2 to v21.1**, you must manually change the status of any `decommissioning` nodes to `decommissioned`. To do this, [run `cockroach node decommission`](remove-nodes.html#step-2-start-the-decommissioning-process-on-the-nodes) on these nodes and confirm that they update to `decommissioned`.
+1. To check the status of decommissioned nodes, run  the [`cockroach node status --decommission`](node-shutdown.html?filters=decommission#cockroach-node-status) command:
 
-In case a decommissioning process is hung, [recommission](remove-nodes.html#recommission-nodes) and then [decommission those nodes](remove-nodes.html#remove-multiple-nodes) again, and confirm that they update to `decommissioned`.
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach node status --decommission
```

Comment:
Having the copy button here suggests that someone can paste this without specifying certs or host. I think it's best to add those parameters (like in other commands on this page).

```suggestion
    cockroach node status --decommission --certs-dir=certs --host={address of any live node}
```

Reply:
I edited this suggestion because it doesn't have to be a live node. For example you can decommission a node after it experiences an unrecoverable hardware failure. It's for the cluster, not for the node.